src/App.tsx

```diff
@@ -1,27 +1,66 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import 'bulma';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
+import { get5First, getAll, getRedGoods } from './api/goods';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const getData = async (method:string) => {
+    let data;
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+    switch (method) {
+      case 'all':
+        data = await getAll();
+        setGoods(data);
```

Comment:
don't repeat yourself: you call `setGoods` after each api request
you can move this call after the `switch \ case` and call it only once